e2e/validation.go

```diff
@@ -39,35 +36,12 @@ func validateWasm(ctx context.Context, t *testing.T, kube *Kubeclient, nodeName,
 
 	execResult, err := pollExecOnPod(ctx, t, kube, defaultNamespace, debugPodName, getWasmCurlCommand(fmt.Sprintf("http://%s/hello", spinPodIP)))
```

Comment:
I'm actually not sure what we want to test here.

Wouldn't waiting for pod containers to be in ready state be enough like we do for nginx?